lib/support/video/scripts/pullNetLogAndroid.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const Promise = require('bluebird');
+const fs = require('fs');
+
+module.exports = {
+    run(context) {
+        const taskData = context.taskData;
+        context.log.debug('Pull netlog');
+        const file = `/sdcard/chromeNetlog-${ context.index}.json`
```

Comment:
This looks like a closed world assumption to me. Can we be sure the netlog always appears there?